src/src/com/microsoft/aad/adal/DateTimeAdapter.java

```diff
@@ -50,12 +50,31 @@
             DateFormat.DEFAULT);
 
     private final DateFormat iso8601Format = buildIso8601Format();
```

Comment:
I mean all dates for tokens in cache should have this format yyyy-MM-dd'T'HH:mm:ss'Z' (because we serialize them with this format only)
How do we get tokens with date format like "MMM dd, yyyy HH:mm:ss" ?